ChangeLog.d/mbedtls_ssl_cert_cb.txt

```diff
@@ -0,0 +1,4 @@
+Features
+   * Add server certificate selection callback near end of Client Hello
+   * Provide mechanism to reset handshake cert list
+   * Accessor to retrieve SNI during handshake, e.g. from cert callback
```

Comment:
Same: please mention the name of the function. (Also, if we agree that it's only from the cert callback, this will need rewording.)

Reply:
:rocket: